### PR TITLE
updated ambiguous description of jquery waypoint

### LINKED ISSUE #2955: Waypoint "Target a Specific Child of an Element Using jQuery" has ambiguous instructions
Link to waypoint: http://freecodecamp.com/challenges/waypoint-target-a-specific-child-of-an-element-using-jquery

There have been quite a few people asking for help regarding this waypoint in the Help channel.

It turns out that their code doesn't work because instead of targeting the `nth-child` of the `target` class, they are targeting the `nth-child` of the `well` class or the `left-well`/`right-well` IDs. The reason for this seems to be the previous waypoint, where the campers target the parent of the `left-well` ID.

The code sample in the waypoint's description uses the `target` class, but I believe that this is not clear enough. The test cases also do not mention the `target` class (but rather the `well` class). As a camper, I myself struggled with this for quite a while before I figured out that I had to use the `target` class.

 The description of the waypoint could be edited to explicitly state that the user must use the `target` class. The test cases must also be edited to mention the `target` class.

### PR DESCRIPTION
**Pull request by butterthebuddha**
closes #2955

### CODE REVIEW

File: seed/challenges/jquery.json [hunk 1]
```diff
@@ -733,11 +733,11 @@
         "You've seen why id attributes are so convenient for targeting with jQuery selectors. But you won't always have such neat ids to work with.",
         "Fortunately, jQuery has some other tricks for targeting the right elements.",
         "jQuery uses CSS Selectors to target elements. <code>target:nth-child(n)</code> css selector allows you to select all the nth element with the target class or element type.",
-        "Make the second child in each of your well elements bounce.",
+        "Make the second child in each of your well elements bounce. You must target the children of the <code>target</code> (NOT THE <code>well</code>) class.",
```
Comment (abhisekp) [inline seed/challenges/jquery.json]: @awesomeaniruddh that's too explicit in specifying what to do and what not to do. :P

Comment (awesomeaniruddh) [inline seed/challenges/jquery.json]: I guess, but this is a way point, not a bonfire. Shouldn't it be fine?

Comment (abhisekp) [inline seed/challenges/jquery.json]: @awesomeaniruddh Maybe. Let's cross our fingers and hope for the best. :smiley:

**Comment by BerkeleyTrue** [comment]
Your PR has a lot of commits mixed in. You may want to copy paste these changes onto a new PR.

**Comment by abhisekp** [comment]
@awesomeaniruddh you may want to use the rebase command instead of merge to reduce the clutter.

**Comment by awesomeaniruddh** [comment]
Fixed it

**Comment by bugron** [comment]
@awesomeaniruddh everything looks good except "NOT THE".

**Comment by awesomeaniruddh** [comment]
Okay, removed it

**Comment by BerkeleyTrue** [comment]
Just ran through this challenge and it is confusing. I believe this update should be a big improvements for our campers. Thanks @awesomeaniruddh!

Happy Coding!

**Comment by BerkeleyTrue** [comment]
@awesomeaniruddh In the future please name branches using the convention `fix/something-to-fix` or `feature/something-new`. See updated [contributing readme](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md).

**Pull request merged**